view/preview/index.css

```diff
@@ -0,0 +1,65 @@
+.preview {
+  position: relative;
+}
+
+.preview__graphics {
+  height: 200vh;
+}
+
+.preview__scroller {
+  position: sticky;
+  transform: scale(0.2);
+  transform-origin: 50% 50vh;
+  opacity: 0;
+  animation: fadeIn 1s cubic-bezier(0, 0.75, 0.25, 1) forwards;
+  will-change: all;
+  transform: translateZ(0);
+  top: 0;
+}
+
+.preview__scroller:before {
+  content: '';
+  background-image: url(./logo.svg);
+  background-repeat: no-repeat;
+  background-size: 50vh 50vh;
+  background-position: center 25vh; 
+  transform-origin: 50% 50vh;
+  transform: scale(0.6);
+  opacity: 0;
+  animation: fadeIn-logo 1s cubic-bezier(0, 0.75, 0.25, 1) forwards;
+  animation-delay: 500ms;
+  will-change: all;
```

Comment:
Maybe trigger it at the moment of an interactive action (scroll)?

Does `will-change` help rather than harm in this form?

Reply:
removed